03-Bash/find_dup_files/find_dup_files.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+path=$(pwd)
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+    echo "$0 to search for files with the same md5  checksum"
+    echo "Usage: $0 [OPTION] or [PATH]"
+    echo "-h, --help     display this help and exit"
+    exit 0
+fi
+
+if [ -n "$1" ]; then
+    path="$1"
+fi
+
+echo "$path"
+
+touch list_files
```

Comment:
The name for temporary file is not unique for each program run.

Reply:
Fixed. Add use mktemp